rollup.config.js

```diff
@@ -14,7 +14,7 @@ export default {
         {dest: 'dist/glamorous.es.js', format: 'es'},
         {dest: 'dist/glamorous.cjs.js', format: 'cjs'},
   ],
-  exports: 'default',
+  exports: 'named',
```

Comment:
I think that I want the UMD global to be the `glamorous` function with `glamorous.ThemeProvider`. I don't want `glamorous.default`. Is this possible? Same thing with the CJS export.